atm-locator/config/db.js

```diff
@@ -24,33 +24,6 @@ const connectDB = async () => {
       const conn = await mongoose.connect(
         `mongodb://${process.env.DATABASE_HOST}:27017/`
```

Comment:
At some point, make all these ports ex 27017 configurable with env parameters. In some customer env, there may be port conflicts with your user range.